magento/magento-functional-tests-migration
# Convert ShoppingCartWithFreeShippingTest to MFTF #394

### Description
This PR checks if Cart Price Rule for free shipping based on total weight works correctly
## Preconditions:
1. Cart Price Rule was created.
## Test flow:
 1. Go to storefront
 2. Add first product with weight<weight specified in rule to shopping cart
 3. Go to shopping cart page
 4. Check if free shipping is not available
 5. Add other product to shopping cart (product1 weight+product2 weight>=weight specified in rule)
 6. Go to shopping cart page
 7. Check if free shipping is available
### Fixed Issues (if relevant)

1.  #394 Convert ShoppingCartWithFreeShippingTest to MFTF
